WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -279,3 +279,25 @@ extension WooAnalyticsEvent {
         }
     }
 }
+
+// MARK: - Variations
```

Comment:
nit: Update this comment to refer to add-ons.

```suggestion
// MARK: - Order Detail Add-Ons
```